2do_fe/src/components/PopUps/AddTask.js

```diff
@@ -1,24 +1,165 @@
-import React from "react";
-import AddTaskPopUp, { openTaskDesc } from "./AddTaskPopUp";
+import React, { useState } from "react";
 import AddTaskIcon from "./AddTaskIcon.svg";
+import PopUp from "./PopUp";
+import logo from "../../logo_2do.png";
+import "./AddTask.css";
+import TextField from "@mui/material/TextField";
+import { useDispatch } from "react-redux";
+import taskActions from "../../redux/actions/task.actions";
+
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { DesktopDatePicker } from '@mui/x-date-pickers/DesktopDatePicker';
+
+function TaskDescForm(props) {
+  
+  const [taskData, setTaskData] = useState({
+    status: "in progress",
+    title: "",
+    description: "",
+    dueDate: "",
+  });
+
+  const [dateValue, setDate] = useState(null);
+
+  const dispatch = useDispatch();
+
+  const handleDateChange = (newDateValue) => {
+    setDate(newDateValue);
+    setTaskData({
+      ...taskData, 
+      dueDate: newDateValue,});
+  };
+
+  const handleChange = (e) => {
+    setTaskData({ 
+      ...taskData, [e.target.name]: e.target.value,
+     });
+  };
+
+  const handleSubmit = () => {
+    //e.preventDefault();
```

Comment:
add e.preventDefault() and e in the function arg

Reply:
Yep added